### PR TITLE
MAINT: Add infant dataset

### PR DESCRIPTION
**Pull request by larsoner**
### Before merging …

- [x] Changelog has been updated (`docs/source/changes.md`)

Locally `python tests/run_tests.py --download=1 ds004229` works, let's see if CircleCI agrees!

https://openneuro.org/datasets/ds004229/

### CODE REVIEW

File: tests/configs/config_ds004229.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+"""
+Single-subject infant dataset for testing maxwell_filter with movecomp.
+
+https://openneuro.org/datasets/ds004229
+"""
+study_name = 'amnoise'
+bids_root = '~/mne_data/ds004229'
+deriv_root = '~/mne_data/derivatives/mne-bids-pipeline/ds004229'
+
+task = 'amnoise'
```
Comment (larsoner) [inline tests/configs/config_ds004229.py:10]:
```suggestion
task = 'amnoise'
crop_runs = (0., 300.)
```

**Comment by larsoner** [comment]
All good only takes ~1.5 min to process (plus download time):

https://app.circleci.com/pipelines/github/mne-tools/mne-bids-pipeline/3076/workflows/81cc64db-e05c-4c3a-a735-958f1b35f2fe/jobs/30662

Feel free to review/merge @hoechenberger @agramfort

**Comment by larsoner** [comment]
... this is a step in https://github.com/mne-tools/mne-bids-pipeline/issues/574

**Review by agramfort — APPROVED** [review]
all good https://output.circle-artifacts.com/output/job/7669ca00-5513-4742-901a-f77b69be2aea/artifacts/0/site/examples/ds004229.html#generated-output !

**Comment by agramfort** [comment]
thx @larsoner

**Pull request merged**